src/components/PupilComponents/CopyrightNotice/CopyrightNotice.tsx

```diff
@@ -0,0 +1,56 @@
+import Link from "next/link";
+import { OakLink, OakP } from "@oaknational/oak-components";
+
+import { resolveOakHref } from "@/common-lib/urls";
+
+type CopyrightNoticeProps = {
+  isLegacyLicense: boolean;
+};
+
+/**
+ * Displays the copyright notice for a lesson.
+ * `isLegacyLicense` indicates whether the lesson is using the current or old license.
+ */
+export const CopyrightNotice = ({ isLegacyLicense }: CopyrightNoticeProps) => {
+  if (isLegacyLicense) {
+    return (
+      <OakP $font="body-4" $color="text-subdued">
+        This content is made available by Oak National Academy Limited and its
+        partners and licensed under{" "}
+        <OakLink
+          element={Link}
+          href={resolveOakHref({
+            page: "legal",
+            legalSlug: "terms-and-conditions",
+          })}
+        >
+          Oak's terms & conditions
+        </OakLink>{" "}
+        (Collection 1), except where otherwise stated.
+      </OakP>
+    );
+  }
+
+  return (
+    <OakP $font="body-4" $color="text-subdued">
+      This content is © Oak National Academy Limited (2023), licensed on{" "}
+      <OakLink
+        href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+        rel="external"
+      >
```

Comment:
The link for the OGL doesn't automatically open in a new tab. I can of course make it do that but personally I trust users to know how to do it themselves or to be able to use the back button. This is the hill I will die on.